OpenRA.Game/Network/Order.cs

```diff
@@ -12,26 +12,21 @@
 using System.IO;
 using System.Linq;
 using OpenRA.Network;
+using ProtoBuf;
 
 namespace OpenRA
 {
-	[Flags]
-	enum OrderFields : byte
+	[ProtoContract]
+	sealed class LegacyOrder
```

Comment:
It would be better to keep this named `Order` until/unless somebody actively starts migrating away from these.